src/MainContent/FlowerListFilters.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import { Col, Container, Form, Row } from "react-bootstrap";
+import DropdownButton from "react-bootstrap/DropdownButton";
```

Comment:
'Dropdown' is declared but its value is never read, taip pat Col, Row - turbūt gali ištrinti unused importus?